### PR TITLE
Ta rh implementing meetings

### PR DESCRIPTION
**Pull request by rezahusain**
Hey guys! So this is what Young implemented along with some database changes and refactoring which I did. After this PR gets merged she'll clean up the look and make it fully functional. Let me know if you guys have any questions.

### CODE REVIEW

**Review by YoungSong99 — APPROVED** [review]
Thanks Reze. I will update the frontend view 👍

File: app/views/shared/_navbar.html.erb
Comment (brvarner) [inline app/views/shared/_navbar.html.erb:1]: like this link addition!

**Review by brvarner — APPROVED** [review]
looks good!

**Review by MayaS1111 — APPROVED** [review]
Great can't wait to see it all done!

**Review by APatel-AI — APPROVED** [review]
Looks good!

**Pull request merged**